src/pages/authentication/signup/Signup.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react";
+import {
+  FaEnvelope,
+  BsFillEyeFill,
+  BsFillEyeSlashFill,
+  BsArrowRight,
+} from "../../../assets/icons";
+import { Signup as SignupImage } from "../../../assets/images";
+import { Link, useNavigate } from "react-router-dom";
+import "../authentication.css";
+import {useAuth} from "../../../context/auth/AuthContext";
+
```

Comment:
Same here you can import all files in single file

Reply:
rightly pointed out by @Ankur9669